modular_skyrat/modules/modular_disabler/kits.dm

```diff
@@ -0,0 +1,19 @@
+/obj/item/device/custom_kit/disabler
+	var/ammo_to_add
+	var/name_to_append
+
+/obj/item/device/custom_kit/disabler/afterattack(obj/target, mob/user, proximity_flag)
+	var/obj/item/gun/energy/disabler/upgraded/target_obj = target
```

Comment:
typecast the `target` argument to `/obj/item/gun/energy/disabler/upgraded`, do an `istype(target)` check here with an early return instead